packages/server/src/tests/utilities/api/base.ts

```diff
@@ -107,26 +109,29 @@ export abstract class TestAPI {
     const headersFn = publicUser
       ? this.config.publicHeaders.bind(this.config)
       : this.config.defaultHeaders.bind(this.config)
-    let request = this.request[method](url).set(
+
+    const app = getServer()
+    let req = request(app)[method](url)
```

Comment:
This change serves two purposes:

1. It removes another dependency on `TestConfiguration`.
2. It's an attempt to remove flakiness, though admittedly a hail-mary. `supertest`, in all of their examples, create a new `request(app)` instance for every test. I'm wondering if this is deliberate, and `supertest` isn't actually safe to reuse, and that's causing some of the very odd failures we see. Haven't been able to prove it, but the runtime of doing this didn't seem different to "caching" the supertest instance, so I'd like to leave it in.

Reply:
I feel like this is worth a try, its very possibly causing problems re-using the same request and I doubt this will change performance of the tests much at all - if its even recognisable.